downburst/create.py

```diff
@@ -205,6 +207,23 @@ def create(args):
         )
     dom = conn.defineXML(etree.tostring(domainxml))
     dom.create()
+    try:
+        env = os.environ
+        pid = os.getpid()
+        # os.getppid() wont return the correct value:
+        ppid = open('/proc/{pid}/stat'.format(pid=pid)).read().split()[3]
+        ppcmdline = open('/proc/{ppid}/cmdline'.format(ppid=ppid)).read().split(b'\x00')
+
+    except IndexError, IOError:
+        log.error('Something went wrong getting PPID/cmdlineinfo')
```

Comment:
`log.error` will not show the exception and the exception is not being used to log what went wrong. Consider adding (or using) `log.exception` and/or using the exception object to log what went wrong